README.md

```diff
@@ -6,27 +6,106 @@
     <img src="https://codecov.io/gh/awslabs/amazon-s3-find-and-forget/branch/master/graph/badge.svg">
 </h1>
 
-> Warning: This project is currently in beta and should not be used for
-> production purposes.
-
 Amazon S3 Find and Forget is a solution to the need to selectively erase records
 from data lakes stored on Amazon Simple Storage Service (Amazon S3). This
 solution can assist data lake operators to handle data erasure requests, for
 example, pursuant to the European General Data Protection Regulation (GDPR).
 
-The Amazon S3 Find and Forget solution can be used with Parquet-format data
-stored in Amazon S3 buckets. Your data lake is connected to the solution via an
-AWS Glue table and specifying which columns in the table contain user
-identifiers.
+The solution can be used with Parquet-format data stored in Amazon S3 buckets.
+Your data lake is connected to the solution via AWS Glue tables and by
+specifying which columns in the tables contain user identifiers.
 
 Once configured, you can queue record identifiers that you want the
 corresponding data erased for. You can then run a deletion job to remove the
 data corresponding to the records specified from the objects in the data lake. A
 report log is provided of all the S3 objects modified.
 
+## Installation
+
+The solution is available as an AWS CloudFormation template and should take
+about 20 to 40 minutes to deploy. See the
+[deployment guide](docs/USER_GUIDE.md#deploying-the-solution) to launch the
+deployment with just one click and the
+[cost overview guide](docs/COST_OVERVIEW.md) to learn about its costs.
+
+> Warning: This project is currently in beta release
+>
+> - Actively developed by AWS
+> - Carefully review code and documentation if you intend to use the solution
+>   for production purposes
+> - Get in touch with the team via
+>   [Github Issues](https://github.com/awslabs/amazon-s3-find-and-forget/issues)
+>   for providing feedback and to ask any question. We appreciate your help
+
+## Usage
+
 The solution provides a web user interface, and a REST API to allow you to
 integrate it in your own applications.
 
+![HomePage Screenshot](docs/images/home-screenshot.png)
+
+See the [user guide](docs/USER_GUIDE.md) to learn how to use the solution and
+the [API Specification](docs/api/README.md) to integrate the solution to your
+own applications.
+
+## Architecture
+
+The goal of the solution is to provide a secure, reliable, performant and cost
+effective tool for finding and removing individual records within objects stored
+in S3 buckets. In order to achieve these goals the solution has adopted the
+following design principles:
```

Comment:
The design principles in the architecture page are actually a bit of a mixture of design principles and consequences of some of our design principles. I tried to re-organise them here. If this makes sense, then I can also copy and paste there.